source/draw/Map.ooc

```diff
@@ -8,9 +8,73 @@
 
 use geometry
 
+_MapEntry: class {
```

Comment:
A bit overkill to have a new class for each entry when we already have cells around each value. Can this inherit from Cell?